src/models/languages.js

```
@@ -0,0 +1,23 @@
+const { Schema, model } = require('mongoose')
+const { LANGUAGE } = require('~/consts/models')
+
+const languageSchema = new Schema(
+  {
+    _id: {
+      type: String,
+      required: [true, "Field '_id' cannot be empty"]
+    },
+    name: {
+      type: String,
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)